add2cal/add2cal.py

```diff
@@ -116,6 +121,10 @@ def ical_content(self):
         c = Calendar()
         e = Event()
         e.alarms = [DisplayAlarm(trigger=self.trigger_datetime)]
+        e.attendees = [Attendee(rsvp=False, role="REQ-PARTICIPANT",
```

Comment:
The library provides an "add_attendee" method. I also discovered that the "rsvp=False" attribute being passed into the Attendee object causes some exception. I think we can remove the rsvp attribute since that is set to false by default. this line should look like:

```
if self.attendee_email:
    e.add_attendee(Attendee(role="REQ-PARTICIPANT",partstat="ACCEPTED", cutype="INDIVIDUAL",email=self.attendee_email,common_name=self.attendee_name))
```